README.md

```diff
@@ -31,6 +31,15 @@ listening for new block events.
 Once started the application will start syncing the blockchain and
 loading blocks into the attached database.
 
+### Installing Ubuntu Required Packages
+
+If running on Ubuntu, you will need the following packages installed before
+running `make release`:
+
+```bash
+sudo apt install erlang cmake libsodium-dev libssl-dev
```

Comment:
@chrisabruce did you now have to add an ESL apt repository to install erlang?

Reply:
nope.  Ubuntu 19.04

Reply:
but what version of erlang did it actually install?

Reply:
22